code/modules/mob/living/simple_animal/hostile/mining_mobs/brimdemon.dm

```diff
@@ -0,0 +1,189 @@
+	playsound(src, 'sound/creatures/brimdemon.ogg', 150, FALSE, 0, 3)
+	cut_overlay("brimdemon_telegraph_dir")
+	var/turf/target_turf = get_ranged_target_turf(src, dir, BRIMBEAM_RANGE)
+	var/turf/origin_turf = get_turf(src)
+	var/list/affected_turfs = get_line(origin_turf, target_turf) - origin_turf
+	for(var/turf/affected_turf in affected_turfs)
+		var/blocked = FALSE
+		if(affected_turf.opacity)
+			blocked = TRUE
+		for(var/obj/potential_block in affected_turf.contents)
+			if(potential_block.opacity)
+				blocked = TRUE
+				break
+		if(blocked)
+			break
+		var/atom/new_brimbeam = new /obj/effect/brimbeam(affected_turf)
+		new_brimbeam.dir = dir
+		beamparts += new_brimbeam
+		for(var/mob/living/hit_mob in affected_turf.contents)
+			if(istype(hit_mob, /mob/living/simple_animal/hostile/asteroid/brimdemon))
+				continue
+			hit_mob.adjustFireLoss(25)
+			to_chat(hit_mob, "<span class='userdanger'>You're hit by [src]'s brimbeam!</span>")
+	if(length(beamparts))
+		var/atom/last_brimbeam = beamparts[length(beamparts)]
+		last_brimbeam.icon_state = "brimbeam_end"
+		var/atom/first_brimbeam = beamparts[1]
+		first_brimbeam.icon_state = "brimbeam_start"
+	addtimer(CALLBACK(src, .proc/end_laser), 2 SECONDS)
+
+/mob/living/simple_animal/hostile/asteroid/brimdemon/proc/end_laser()
+	if(stat != DEAD)
+		icon_state = initial(icon_state)
+	move_resist = initial(move_resist)
+	set_dir_on_move = initial(set_dir_on_move)
+	firing = FALSE
+	for(var/obj/effect/brimbeam/beam in beamparts)
+		animate(beam, time = 0.5 SECONDS, alpha = 0)
+		QDEL_IN(beam, 0.5 SECONDS)
+		beamparts -= beam
+
+/obj/effect/brimbeam
+	name = "brimbeam"
+	icon = 'icons/mob/brimdemon.dmi'
+	icon_state = "brimbeam_mid"
+	layer = ABOVE_MOB_LAYER
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	light_color = LIGHT_COLOR_BLOOD_MAGIC
+	light_power = 3
+	light_range = 2
+
+/obj/effect/brimbeam/Initialize()
+	. = ..()
+	START_PROCESSING(SSfastprocess, src)
+
+/obj/effect/brimbeam/Destroy()
+	STOP_PROCESSING(SSfastprocess, src)
+	return ..()
+
+/obj/effect/brimbeam/process()
+	for(var/mob/living/hit_mob in get_turf(src))
+		damage(hit_mob)
+
+/obj/effect/brimbeam/proc/damage(mob/living/hit_mob)
+	if(istype(hit_mob, /mob/living/simple_animal/hostile/asteroid/brimdemon))
+		return
+	hit_mob.adjustFireLoss(5)
+	to_chat(hit_mob, "<span class='danger'>You're damaged by [src]!</span>")
+
+/obj/item/crusher_trophy/brimdemon_fang
+	name = "brimdemon's fang"
+	icon_state = "brimdemon_fang"
+	desc = "A fang from a brimdemon's corpse."
+	denied_type = /obj/item/crusher_trophy/brimdemon_fang
+	var/static/list/comic_phrases = list("BOOM", "BANG", "KABLOW", "KAPOW", "OUCH", "BAM", "KAPOW", "WHAM", "POW", "KABOOM")
+
+/obj/item/crusher_trophy/brimdemon_fang/effect_desc()
+	return "mark detonation creates effects on the target"
+
+/obj/item/crusher_trophy/brimdemon_fang/on_mark_detonation(mob/living/target, mob/living/user)
+	target.balloon_alert_to_viewers("[pick(comic_phrases)]!")
```

Comment:
Why not just say?

Reply:
because they arent saying it its a visual effect